File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] ans = new int[nums.length * 2];

        int count = 0;

        for(int i = 0; i < ans.length; i++)
        {
            if(i < nums.length)
            {
                ans[i] = nums[i];
            }
            else
            {
                ans[i] = nums[count];
                count++;
            }
            
        }
        return ans;
    }
}

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        heap = []
        
        #min heap, pop will remove smaller values
        for val in nums:
            heapq.heappush(heap, val)
            
            if len(heap) > k:
                heapq.heappop(heap)
            
        return heap[0]
